recipe/meta.yaml

```diff
@@ -19,14 +19,14 @@ build:
     - lib/R/lib/
     - lib/
 
-requirements:  # [not win32]
-  build:  # [not win32]
-    - r-base  # [not win32]
-    - posix  # [win64]
-    - m2w64-toolchain  # [win64]
-    - gcc  # [not win]
```

Comment:
That is at least a different story. Also, I am not convinced it is a hack. Those are different dependencies for different platforms. It is better to explicitly see which dependency occurs in which platform. My fix is just to so that the ridiculous selectors on the whole section are handled by build-skip instead.